### PR TITLE
Add support for Playnite game logo and video files

### PR DESCRIPTION
**Pull request by polson**
See https://github.com/mmatyas/pegasus-frontend/issues/874

### CODE REVIEW

File: src/backend/providers/playnite/PlayniteProvider.cpp [hunk 1]
```diff
@@ -114,6 +114,17 @@ void apply_game_fields(
             QStringLiteral("%1/library/files/%2").arg(playnite_dir_path, game_info.background_image);
         game.assetsMut().add_file(AssetType::BACKGROUND, bg_path);
     }
+
+    const QString extra_metadata_path = QStringLiteral("%1/ExtraMetadata/games/%2").arg(playnite_dir_path, game_info.id);
```
Comment (mmatyas) [inline src/backend/providers/playnite/PlayniteProvider.cpp:118]: `game_info.id` could be empty here.

**Review by mmatyas — COMMENTED** [review]
Thanks! Found just one thing, not sure if that's intentional.

File: src/backend/providers/playnite/PlayniteProvider.cpp (hunk 1 above)
Comment (polson) [reply src/backend/providers/playnite/PlayniteProvider.cpp:118]: Good catch, added validation for game ID

**Comment by mmatyas** [comment]
Looks good!

**Pull request merged**